exasol/driver/websocket/_cursor.py

```diff
@@ -239,6 +291,12 @@ def executemany(self, operation, seq_of_parameters):
         ]
         connection = self._connection.connection
         self._cursor = connection.cls_statement(connection, operation, prepare=True)
+
+        parameter_data = self._cursor.parameter_data
```

Comment:
do you need a new local variable here?

Reply:
this is just to have better shorter access (to improve readability)